src/coreclr/src/pal/src/locale/unicode.cpp

```diff
@@ -260,70 +260,6 @@ GetACP(VOID)
     return CP_UTF8;
 }
 
-
-/*++
-Function:
-IsDBCSLeadByteEx
-
-See MSDN doc.
---*/
-BOOL
-PALAPI
-IsDBCSLeadByteEx(
-     IN UINT CodePage,
-     IN BYTE TestChar)
-{
-    CPINFO cpinfo;
-    SIZE_T i;
-    BOOL bRet = FALSE;
-
-    PERF_ENTRY(IsDBCSLeadByteEx);
-    ENTRY("IsDBCSLeadByteEx(CodePage=%#x, TestChar=%d)\n", CodePage, TestChar);
-
-    /* Get the lead byte info with respect to the given codepage*/
-    if( !GetCPInfo( CodePage, &cpinfo ) )
```

Comment:
There are two usages of GetCPInfo left in PAL:
https://github.com/dotnet/runtime/blob/513ade6f40cde36c2663f0fbd6c8b0947439ba28/src/coreclr/src/pal/src/cruntime/mbstring.cpp#L126
https://github.com/dotnet/runtime/blob/513ade6f40cde36c2663f0fbd6c8b0947439ba28/src/coreclr/src/pal/src/cruntime/mbstring.cpp#L183

but logically these conditions will never satisfied as we have hardcoded MaxCharSize=4 in success case.

Reply:
Agree. I would delete `GetCPInfo` and all its uses.